pyalex/api.py

```diff
@@ -191,13 +193,16 @@ def get(self, return_meta=False, page=None, per_page=None, cursor=None):
                 l.append(k + "=" + quote_plus(str(v)))
 
         url = self.url + "?" + "&".join(l)
-
+        params = {"api_key": config.api_key} if config.api_key else {}
         res = requests.get(
-            url, headers={"User-Agent": "pyalex/" + __version__, "email": config.email}
+            url,
+            headers={"User-Agent": "pyalex/" + __version__, "email": config.email},
+            params=params
         )
         res_json = res.json()
 
-        if res.status_code == 403 and "query parameters" in res_json["error"]:
+        # Removed "query parameters" in res_json["error"] as res_json["error"] is a boolean
+        if res.status_code == 403:
```

Comment:
```suggestion
        if res.status_code == 403 and "query parameters" in res_json["error"]:
```

I would like to revert this part in favor of the changes in PR #9. OpenAlex sometimes returns an HTML page on an error and sometimes a JSON error. I think pyalex sometimes breaks because of incorrect handling (also on TOO_MANY_REQUESTS). Or do I miss something?